README.md

```diff
@@ -4,8 +4,92 @@ In creating numerous CloudFront distributions across our projects, we found the
 resource to be a bit boilerplate heavy.  This led us to making a wrapper for it that attempts to simplify the resource
 and make configuration a bit more readable.
```

Comment:
A `## Features` block up here would be a good idea to highlight the features we think provide value in this module.